notebooks/19_practical_methods__aac.ipynb

```diff
@@ -0,0 +1,783 @@
+{
```

Comment:
<p><code>gs</code> for graphspace should probably be avoided, since it is the shortcut we use for geomstats_backend </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>